src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -269,7 +269,7 @@ internal abstract partial class AbstractInheritanceMarginService
             }
         }
 
-        return items.ToImmutable();
+        return items.ToImmutableAndClear();
```

Comment:
>items [](http://example.com/codeflow?start=15&length=5)

maybe do this for targetItems both places above too?

Reply:
this one had too many conditionals.

Reply:
Just making sure we're on the same page. My request was to change the two "targetItems" arraybuilders declared inside the outer loop above.

Reply:
the issue is we're dealing with an IGrouping, and don't know how many groups there are.

Reply:
I see, thanks!